### ISSUE TITLE
LibertyBans Incompatibility 

### ISSUE THREAD

**Issue by griffins07** [EncryptSL/CENSOR-RELOADED-ISSUES#27]
People can bypass LibertyBans mutes by swearing in chat and then saying whatever they want.

Paper
1.18.1
4.3.3 (LibertyBans 1.0.0)
Java 17

**Comment by EncryptSL** [comment]
I don't have tested with LibertyBans i must look on this deeper.

**Comment by griffins07** [comment]
Yeah libertybans can't fix it and they've confirmed the issue. Needs to be fixed on your end: https://www.toptal.com/developers/hastebin/kawociqage.css

**Comment by griffins07** [comment]
I'm not sure if it's just a velocity bug either.

**Comment by EncryptSL** [comment]
I can provide testing on my side, probably a bug in plugin.
